src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/config/PluginSettings.java

```diff
@@ -59,6 +63,8 @@
   private Properties settings;
   private final String configFilePath;
 
+  private long batchMetricsRetentionPeriod;
```

Comment:
Please add the timeunit as the variable name

Reply:
Yes